source/precalculus/source/meta/graphs.ptx

```diff
@@ -0,0 +1,17 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<appendix xml:id="graphs-of-common-functions" xmlns:xi="http://www.w3.org/2001/XInclude">
+    <title>Graphs of Common Functions</title>
+    <paragraphs>
+        <title>Polynomial Functions</title>
+            <figure>
+                <image>
+                    <sageplot>
+                        f=x
+                        plot(f,xmin=-5,xmax=5,ymin=-5,ymax=5,thickness=3,aspect_ratio=1,gridlines=True, ticks=[[-5..5],[-5..5]])
```

Comment:
```suggestion
f=x
p = plot(f,xmin=-5,xmax=5,ymin=-5,ymax=5,thickness=3,aspect_ratio=1,gridlines=True, ticks=[[-5..5],[-5..5]])
p += text(r'$f(x)=x$', (-3.5, 3.5), fontsize="x-large")
p
```

Only way I can think of to guarantee to have the function appear in a particular place